python/python/bystro/vcf_utils/tests/test_sim_data_generator.py

```diff
@@ -0,0 +1,35 @@
+"""Test simulated data."""
+
+import pandas as pd
+from bystro.vcf_utils.sim_data_generator import (
+    generate_random_vcf_index, 
+    generate_simulated_vcf
+)
+
+HEADER_COLS = ["#CHROM", "POS", "ID", "REF", "ALT", "QUAL", "FILTER", "INFO", "FORMAT"]
+
+
+def test_generate_random_vcf_index():
+    """Test expected index components have correct parameters."""
+    random_vcf_index = generate_random_vcf_index()
+    assert isinstance(random_vcf_index, str)
+    parts = random_vcf_index.split(":")
+    assert len(parts) == 4
+    assert parts[0].isdigit()
+    assert 1 <= int(parts[0]) <= 22
+    assert parts[1].isdigit()
+    assert 1 <= int(parts[1]) <= 1000000
```

Comment:
nit: probably what we really care about is the fact that it's >= 1, the 1_000_000 is an implementation detail